packages/next/server/api-utils/index.ts

```diff
@@ -71,12 +71,22 @@ export function redirect(
 }
 
 export const PRERENDER_REVALIDATE_HEADER = 'x-prerender-revalidate'
+export const PRERENDER_REVALIDATE_IF_GENERATED_HEADER =
+  'x-prerender-revalidate-if-generated'
```

Comment:
```suggestion
export const PRERENDER_REVALIDATE_ONLY_GENERATED_HEADER =
  'x-prerender-revalidate-only-generated'
```

Perhaps this could be renamed too?

Reply:
Updated the const name although the header has to stay the same unless we change it in the proxy as well

Reply:
I see